code/datums/mind/antag.dm

```diff
@@ -33,13 +33,14 @@
 	var/datum/antagonist/A = has_antag_datum(datum_type)
 	if(A)
 		A.on_removal()
+		log_game("[key_name(src)] has lost antag datum [A.name]([A.type]).")
```

Comment:
```suggestion
		src.log_message("has lost antag datum [A.name]([A.type]).", LOG_GAME)
```

Reply:
I believe that log_message is on mob, so you may have to do src.mob. Am mobileposting so I'm not 100%

Reply:
Yeah it's an atom proc, passed it through current. instead and it seemed to work fine, cheers